website/static/js/project-organizer.js

```diff
@@ -35,22 +35,23 @@ function _poTitleColumn(item) {
         e.stopImmediatePropagation();
     };
     var node = item.data; // Where actual data of the node is
+    var title = $osf.decodeText(node.attributes.title);
     var css = ''; // Keep for future expandability -- Remove: item.data.isSmartFolder ? 'project-smart-folder smart-folder' : '';
     var isMypreprintsCollection = tb.options.currentView().collection.data.nodeType === 'preprints';
     if (item.data.archiving) { // TODO check if this variable will be available
-        return  m('span', {'class': 'registration-archiving'}, node.attributes.title + ' [Archiving]');
+        return m('span', {'class': 'registration-archiving'}, title + ' [Archiving]');
```

Comment:
Not quite sure how these changes made it into this PR. Could you clean out the unrelated changes?